fundingmanager_test.go

```diff
@@ -2391,6 +2420,8 @@ func TestFundingManagerCustomChannelParameters(t *testing.T) {
 // TestFundingManagerMaxPendingChannels checks that trying to open another
 // channel with the same peer when MaxPending channels are pending fails.
 func TestFundingManagerMaxPendingChannels(t *testing.T) {
+	t.Parallel()
```

Comment:
IIRC there's some lingering global variable race condition that prevented us from doing this in the past?

Reply:
I believe that was addressed by https://github.com/lightningnetwork/lnd/pull/3165/commits/fe7d9143d8e15bfa5f57793e52c7110a643faf04.

Reply:
Yep exactly :)